### DISCUSSION TITLE
Flash-card and Topic-wise tag based questions filter

### DISCUSSION THREAD

**Discussion by adilwahla** [ykdojo/defaang#72] [Ideas]
### Hi! Yk and Dear Defaang Family 
-I would like to introduce tag based filter questions what are your thoughts?
-I would like to add flip card for DSA one side for code & other side for algo 

what are your thoughts @ykdojo 
Thanks in advance

**Comment by ykdojo** [comment]
Can you clarify your question a bit more? I don't really quite understand it.

The discussion on #59 might be relevant, too.

**Reply by adilwahla** [reply]
I mean, while learning DSA soon newbie start forgetting things so, i want to add  a say flash-card which will help them recalling & saving algorithm & code on side by side (can be saved for future recall purpose) 
Secondly, topic-wise questions could be filtered out let say i covered loops & all questions relevant to loop will show up to solve

**Reply by ykdojo** [reply]
Right, having topics might be helpful in the future. It's probably too much to add in the beginning, though.

I think flashcards would be outside the scope of this project for a long time.

**Reply by adilwahla** [reply]
Cool i will share shortly mock-up on how topics could make Defaang nicer & super cool
